docs/cluster-deployment/stripe.rst

```diff
@@ -0,0 +1,164 @@
+.. _cluster-deployment-stripe:
+
+
+===============================
+Deploy a cluster by credit card
```

Comment:
Perhaps we could make this a bit more generic? "Deploying by credit card" sounds a bit strange! :) Maybe just "Deploying a cluster"? If this is the default option, then the azure/aws ones can specify the qualifiers.

Reply:
There is a point to that. I was worried it might cause confusion, but I agree it has the benefit of making it even more strongly the default suggestion.